src/openseadragon.js

```diff
@@ -659,6 +659,8 @@
   * @property {Object} [ajaxHeaders={}]
   *     A set of headers to include when making AJAX requests for tile sources or tiles.
   *
+  * @property {Boolean} [allowPost=false]
```

Comment:
This is a pretty specific option, so it should have a longer, more explicit name. Something like `acceptHashDataForPost` or so. 

Also it might be good to add a little more detail in the comment here, like that this refers to URLs, and which URLs (tile source/tiles), and that the expected format is ampersand-separated. This information already exists in other places, but this is a nice central location to provide an overview.